driver-core/src/main/com/mongodb/client/model/changestream/ChangeStreamDocument.java

```diff
@@ -111,8 +142,31 @@ public BsonDocument getResumeToken() {
      *
      * @return the namespace
```

Comment:
Let's improved the Javadoc to indicate which operation types don't have a MongoNamespace, and what the exception is that will be thrown when there is no namespace

Reply:
Done